### PR TITLE
GUI: Label fixups

### PR DESCRIPTION
**Pull request by luke-jr**
Note: This needs adapting if #5861 gets merged first. git will NOT detect the conflict.

### CODE REVIEW

File: src/qt/peertablemodel.cpp [hunk 1]
```diff
@@ -115,7 +115,7 @@ PeerTableModel::PeerTableModel(ClientModel *parent) :
     clientModel(parent),
     timer(0)
 {
-    columns << tr("Address/Hostname") << tr("User Agent") << tr("Ping Time");
+    columns << tr("Node/Service") << tr("User Agent") << tr("Ping Time");
```
Comment (Diapolo) [inline src/qt/peertablemodel.cpp]: Why are you changing this?

File: src/qt/bitcoingui.cpp [hunk 2]
```diff
@@ -859,18 +859,19 @@ void BitcoinGUI::closeEvent(QCloseEvent *event)
...
-void BitcoinGUI::incomingTransaction(const QString& date, int unit, const CAmount& amount, const QString& type, const QString& address)
+void BitcoinGUI::incomingTransaction(const QString& date, int unit, const CAmount& amount, const QString& type, const QString& address, const QString& label)
 {
     // On new transaction, make an info balloon
+    QString msg = tr("Date: %1\n").arg(date) +
+                  tr("Amount: %1\n").arg(BitcoinUnits::formatWithUnit(unit, amount, true));
+                  tr("Type: %1\n").arg(type);
+    if (!label.isEmpty())
+        msg += tr("Label: %1\n").arg(label);
+    else
```
Comment (Diapolo) [inline src/qt/bitcoingui.cpp]: Nit: ``else if``

File: src/qt/peertablemodel.cpp (hunk 1 above)
Comment (luke-jr) [inline src/qt/peertablemodel.cpp]: "Address" can be easily confused with bitcoin addresses, and it never displays a hostname. It does, however, display the service (port).

File: src/qt/bitcoingui.cpp [hunk 3]
```diff
@@ -859,18 +859,19 @@ void BitcoinGUI::closeEvent(QCloseEvent *event)
 }
 
 #ifdef ENABLE_WALLET
-void BitcoinGUI::incomingTransaction(const QString& date, int unit, const CAmount& amount, const QString& type, const QString& address)
+void BitcoinGUI::incomingTransaction(const QString& date, int unit, const CAmount& amount, const QString& type, const QString& address, const QString& label)
 {
     // On new transaction, make an info balloon
+    QString msg = tr("Date: %1\n").arg(date) +
+                  tr("Amount: %1\n").arg(BitcoinUnits::formatWithUnit(unit, amount, true));
+                  tr("Type: %1\n").arg(type);
```
Comment (jonasschnelli) [inline src/qt/bitcoingui.cpp]:
```
qt/bitcoingui.cpp:867:19: warning: ignoring return value of function declared with warn_unused_result attribute [-Wunused-result]
                  tr("Type: %1\n").arg(type);
```

**Comment by jonasschnelli** [comment]
Hmm... i'm not sure about this one (see below).
If one not uses labels, it looks uncommon.

bildschirmfoto-2015-03-06-um-09 59 30

**Comment by luke-jr** [comment]
I'm not sure it makes sense to not use labels in the real world? You wouldn't be able to tell who paid you or for what... or for what you paid.

File: src/qt/transactiontablemodel.cpp [hunk 4]
```diff
@@ -626,7 +626,7 @@ QVariant TransactionTableModel::headerData(int section, Qt::Orientation orientat
             case Watchonly:
                 return tr("Whether or not a watch-only address is involved in this transaction.");
             case ToAddress:
-                return tr("Destination address of transaction.");
+                return tr("User-defined intent/purpose of the transaction.");
```
Comment (laanwj) [inline src/qt/transactiontablemodel.cpp]: This reminds me of philosophy textbooks. Is there a simpler way to word "User-defined intent/purpose"?

Comment (luke-jr) [inline src/qt/transactiontablemodel.cpp]: "User-provided payment request description"?

Comment (laanwj) [inline src/qt/transactiontablemodel.cpp]: What about `Destination label of the transaction`?

Comment (luke-jr) [inline src/qt/transactiontablemodel.cpp]: I imagine someone looking at the tooltip will be wondering *what* a label is, so just repeating it won't really help.

Comment (laanwj) [inline src/qt/transactiontablemodel.cpp]: "A user defined label for the destination of the transaction" ?

Comment (luke-jr) [inline src/qt/transactiontablemodel.cpp]: For received transactions, it generally describes the sender or purpose of payment.

**Pull request merged**